build/Resume_Sawyer.tex

```diff
@@ -18,34 +18,41 @@
 \begin{document}
 
   % -- Objective -- %
-  \begin{rSection}{Objective}
-    I'm looking for meaningful work involved with the organization, preservation, and expansion of knowledge, with particular regard to library resources and academic research. By improving the engines which power human understanding, I seek to expedite human progress long-term.
+  % \begin{rSection}{Objective}
+  %   I'm looking for meaningful work involved with the organization, preservation, and expansion of knowledge, with particular regard to library resources and academic research. By improving the engines which power human understanding, I seek to expedite human progress long-term.
+  % \end{rSection}
+
+  % -- Overview -- %
+  \begin{rSection}{Overview}
+    3+ years of full stack web development experience with a variety of technologies and frameworks
+
+    Interested in information systems, metadata, interoperability, web standards, and research support
   \end{rSection}
 
   % -- Experience -- %
   \begin{rSection}{Experience}
 
-    \begin{rSubsection}{\underline{Boston University Libraries }: Web Developer }{ Sept 2018 -  }{ Michael Ward - Primo / Web Services Manager - mrward@bu.edu }
+    \begin{rSubsection}{\underline{Boston University Libraries }: Web Developer }{ Sept 2018 - present }{ Michael Ward - Web Services Manager - mrward@bu.edu }
 
-      \item Architect, design, implement, document, and maintain the infrastructural components of a new technical foundation at BU Libraries before we build out our team going forward.
+      \item Architect, design, implement, and document a new technical foundation and workflow at BU Libraries built on web standards, open source collaboration, traceability, and modular reusability.
 
-      \item Generate, manage, and maintain open-source components for our websites and customizations of our search interface on npm.
+      \item Create and expand a suite of custom web components, enabling a more consistent, feature-rich, and data-driven user experience across a variety of websites and platforms.
 
     \end{rSubsection}
 
     \begin{rSubsection}{\underline{ITHAKA - JSTOR }: Junior Web Application Developer }{ Jan 2017 - June 2018 }{ Peter Vlahakis - Product Owner - peter.vlahakis@ithaka.org }
 
-      \item Built, maintained, monitored, and improved upon a series of django applications constituting the new JSTOR 'For Librarians' portal within an agile, distributed, continuous deployment AWS environment.
+      \item Contributed over 250 pull requests into a dozen front and backend repositories in six languages.
```

Comment:
nit: "frontend and backend" instead of "front and backend" for increased readability